File: Basic Programming/Dimik oj/C++/প্রোগ্রামিং সমস্যা 32 - [৫২ সমস্যা বই] X এর গুণিতক.cpp
/*                    Name : Arif Khan
                      Judge: Dimik Oj
                      University: Primeasia University
                      problem: প্রোগ্রামিং সমস্যা 32 - [৫২ সমস্যা বই] X এর গুণিতক
                      Difficulty: Easy
                      Time Complexity: 0.4519 সেকেন্ড
                      Problem Link: https://dimikoj.com/problems/32/multiplication-of-x-by
*/

#include <bits/stdc++.h>
using namespace std;
 
int main() {
	long long n, y, x;
	cin >> n;
	while(n--) {
		long long m = 1;
		cin >> x >> y;
        if(x > y) cout << "Invalid!" << endl;
        else{
        	while(m*x <= y) {
        		cout << m*x << endl;
        		m++;
        	}
        }
            cout << endl;
	}
	return 0;
}

File: Basic Programming/Dimik oj/Python3/প্রোগ্রামিং সমস্যা 32 - [৫২ সমস্যা বই] X এর গুণিতক.py
#                      Name : Arif Khan
#                      Judge: Dimik Oj
#                      University: Primeasia University
#                      problem: প্রোগ্রামিং সমস্যা 32 - [৫২ সমস্যা বই] X এর গুণিতক
#                      Difficulty: Easy
#                      Time Complexity: 0.3997 সেকেন্ড
#                      Problem Link: https://dimikoj.com/problems/32/multiplication-of-x-by
#

for _ in range(int(input())):
	n,m = map(int, input().split())
	if n > m:
		print("Invalid!")
	else:
		for i in range(1,m+1):
			if i%n == 0:
				print(i)
	print()